### DISCUSSION TITLE
Did Monika just sing Can You Feel The Sunshine!?

### DISCUSSION THREAD

**Discussion by GitDude82** [Monika-After-Story/MonikaModDev#9436] [general]
So, I'm playing Monika After Story, and Monika starts singing. Totally normal. I always root for her. But I look at the lyrics and see:

"Without you, there's nothing for me to do. Can you feel the sunshine, too? It's comin' through; it makes me feel brand new," 

This seems normal for 5 seconds until I realize this song is from SONIC R!? HUH, WHAT??
Is this normal?

**Comment by Trolli-Lolz** [comment]
yes .  I  had  the  exact  same  reaction .
